_opensearch/install/plugins.md

```diff
@@ -205,56 +205,61 @@ bin/opensearch-plugin install --batch <plugin-name>
 
 ## Available plugins
 
+Major, minor, and patch plugin versions must match OpenSearch major, minor, and patch versions in order to be compatible. For example, plugins versions 2.3.0.x work only with OpenSearch 2.3.0.
```

Comment:
I think we should either change this to `{: .important }` or add it as a warning? I'm afraid someone might miss it as a just a note.

Reply:
Important did not work, so changed to warning.